include/rocksdb/file_system.h

```diff
@@ -171,6 +171,17 @@ class FileSystem {
   // The result of Default() belongs to rocksdb and must never be deleted.
   static std::shared_ptr<FileSystem> Default();
 
+  // See Env::OnDbPathsRegistered.
```

Comment:
Can we do it the other way around? Comment here, and write en Env.h to see it here? FileSystem is the newer API and we should encourage users to implement this instead.